packages/docs/site/docusaurus.config.js

```diff
@@ -33,7 +33,7 @@ const config = {
 	i18n: {
 		defaultLocale: 'en',
 		path: 'i18n',
-		locales: ['en', 'es', 'fr', 'ja', 'pt-br', 'tl'],
+		locales: ['en', 'es', 'fr', 'ja', 'pt-br', 'tl', 'gu'],
 		localeConfigs: {
```

Comment:
On the `localConfigs` property, you will need to add an object with the Gujarati configuration to finish the process:

```
gu: {
    label: 'Gujarati',
    path: 'gu',
},
```